src/_P087_SerialProxy.ino

```diff
@@ -244,19 +245,17 @@ boolean Plugin_087(uint8_t function, struct EventStruct *event, String& string)
 
       if ((nullptr != P087_data)) {
         String cmd      = parseString(string, 1);
-        bool   writeMix = cmd.equals(F("serialproxy_writemix"));
 
-        if (cmd.equals(F("serialproxy_write")) || writeMix) {
-          String param1;
-
-          if (writeMix) {
-            param1 = parseHexTextString(string);
-          } else {
-            param1 = parseStringKeepCase(string, 2, ',', false); // Don't trim off white-space
-          }
-          parseSystemVariables(param1, false);                   // FIXME tonhuisman: Doesn't seem to be needed?
+        if (cmd.equals(F("serialproxy_write"))) {
+          String param1 = parseStringKeepCase(string, 2, ',', false); // Don't trim off white-space
+          parseSystemVariables(param1, false);                        // FIXME tonhuisman: Doesn't seem to be needed?
           P087_data->sendString(param1);
-          addLogMove(LOG_LEVEL_INFO, param1);                    // FIXME tonhuisman: Should we always want to write to the log?
+          addLogMove(LOG_LEVEL_INFO, param1);                         // FIXME tonhuisman: Should we always want to write to the log?
+          success = true;
+        } else
+        if (cmd.equals(F("serialproxy_writemix"))) {
```

Comment:
Why not using the `equals` command in ESPEasy?

Reply:
Ah, will fix that